client/packages/system/src/Item/DetailView/Tabs/Store.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  DetailContainer,
+  DetailInputWithLabelRow,
+  useTranslation,
+  DetailSection,
+  Grid,
+  NumericTextDisplay,
+  CurrencyInput,
+} from '@openmsupply-client/common';
+import { ItemFragment } from '../../api';
+
+export const StoreTab = ({ item }: { item: ItemFragment }) => {
+  const t = useTranslation();
+  const isDisabled = true;
+
+  return (
+    <DetailContainer>
+      <Grid
+        container
+        flex={1}
+        flexDirection="column"
+        style={{ maxWidth: 500 }}
+        gap={4}
+      >
+        <DetailSection title={t('title.pricing')}>
+          <DetailInputWithLabelRow
+            label={t('label.default-sell-price-per-pack')}
+            Input={
+              <CurrencyInput
+                value={item?.itemStoreJoin?.defaultSellPricePerPack}
+                disabled={isDisabled}
+                onChangeNumber={() => {}}
+                width={'100%'}
+              />
+            }
+            DisabledInput={<NumericTextDisplay value={item?.margin} />}
+          />
+        </DetailSection>
+      </Grid>
+
+      <Grid
```

Comment:
Keeping it grid for now when we add other fields in but can remove and use Box with flexin' instead